### DISCUSSION TITLE
Having a hard time understanding

### DISCUSSION THREAD

**Discussion by surgarshorts** [community/community#51416] [Programming Help] [answered]
How many time do I fill out the form to receive currancy to wallet

**Comment by sergeyleschev** [comment, marked as answer]
Could you please provide more details about the specific form and currency in question?

**Reply by surgarshorts** [reply]
In my email I have people sending me links and when I click on them it takes me to a crypto mining page and I just need to know what I need to do to either cash out in US currency as it keeps telling me I need to do

**Reply by surgarshorts** [reply]
What I'm I to do?

**Reply by surgarshorts** [reply]
I really don't understand anything here. Unfamiliar platforms I struggle navigating or understanding.
